10.9-libcxx/stable/main/finkinfo/sci/gdl.info

```diff
@@ -76,6 +76,7 @@ pushd build
 	-DNCURSESDIR=%p \
 	-DNETCDF=ON \
 	-DOPENMP=OFF \
+	-DZLIBDIR=`xcrun --sdk macosx --show-sdk-path`/usr/include \
```

Comment:
Can you add to the commit description what this line is doing?

Reply:
Maybe something like:

    Find zlib with xcrun show-sdk-path for Mojave.

    Also works on.... Or doesn't work before...

    What changed to make this necessary?

Reply:
Done in DescPort in commit be14375 (below).
<pre>Need to define ZLIBDIR using xcrun since 10.14 (XCode 10) since this
does no longer include a directory /usr/include, but needs to find it
in the MacOS SDK. Works also for previous versions, at least back to
Mac OS 10.9.
</pre>